packages/playwright-crawler/src/internals/enqueue-links/click-elements.ts

```diff
@@ -0,0 +1,536 @@
+import type {
+    GlobInput,
+    PseudoUrlInput,
+    RegExpInput,
+    RequestTransform,
+    UrlPatternObject,
+    RequestQueue,
+    RequestOptions,
+} from '@crawlee/browser';
+import {
+    constructGlobObjectsFromGlobs,
+    constructRegExpObjectsFromPseudoUrls,
+    constructRegExpObjectsFromRegExps,
+    createRequests,
+    createRequestOptions,
+} from '@crawlee/browser';
+import log_ from '@apify/log';
+import type { Dictionary, BatchAddRequestsResult } from '@crawlee/types';
```

Comment:
undeclared dependency

(i believe it actually matters for type imports too, as they will be present in the .d.ts files)